pkg/runtime/reconciler.go

```diff
@@ -156,7 +156,7 @@ func (r *resourceReconciler) Reconcile(ctx context.Context, req ctrlrt.Request)
 	roleARN := r.getRoleARN(acctID)
 	endpointURL := r.getEndpointURL(desired)
 	gvk := desired.RuntimeObject().GetObjectKind().GroupVersionKind()
-	sess, err := r.sc.NewSession(region, &endpointURL, roleARN, gvk)
+	sess, err := r.sc.NewSession(region, &endpointURL, r.cfg.ForceS3PathStyle, roleARN, gvk)
```

Comment:
I don't see why we wouldn't add a similar flag if this is supported by aws-sdk-go...
/remove-lifecycle frozen